paimon-common/src/main/java/org/apache/paimon/data/columnar/ColumnarRowIterator.java

```diff
@@ -45,7 +45,6 @@ public ColumnarRowIterator(ColumnarRow rowData, @Nullable Runnable recycler) {
         super(recycler);
         this.rowData = rowData;
         this.recycler = recycler;
-        this.rowPosition = 0;
```

Comment:
currently, we have `pos` and `rowPosition`.

Can we rename them to `nextPos` and `nextGlobalPosition`?